docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/git-commit.md

```diff
@@ -66,7 +67,6 @@ steps:
     message: ${{ outputs['update-image'].commitMessage }}
 # Push, etc...
 ```
-
```

Comment:
Please put this blank line back.

Reply:
Asking for it to be _put back_. Markdown linters don't like this.

Reply:
done.